subsys/net/lib/http/http_server_http1.c

```diff
@@ -403,6 +404,7 @@ int handle_http1_static_fs_resource(struct http_resource_detail_static_fs *stati
 	 * for the content type and encoding
 	 */
 	char http_response[sizeof(RESPONSE_TEMPLATE_STATIC_FS) + HTTP_SERVER_MAX_CONTENT_TYPE_LEN +
+			   sizeof("Content-Length: 01234567890123456789\r\n") +
```

Comment:
would certainly be an interesting experiment to send 99999999999999999999 bytes (86 exabytes!) over HTTP :)

Reply:
We just allocate "enough" space for the temp buffer (in this case a bit more than actually needed). I am not really expecting to send that many bytes.

Reply:
Ya, I know :) I guess this was just a snarky way to say that you indeed definitely allocated "a bit more" digits than what's ever gonna be needed, but it's not like targets running the HTTP server can't afford 6 extra bytes or so